README.md

```diff
@@ -367,6 +367,30 @@ Example:
 }
 ```
 
+### Special GDD Type: "Invoke Action"
+
+Some templates support invoking custom functions during their life-time.
+This GDD Type should be rendered as a Button in the GUI so that the user can click it to invoke the action.
+
+
+Example:
+```typescript
+{
+  "title": string, // [Mandatory] A short title / label of the action
+  "description": "", // [Optional] A longer description of the action
+  "type": "null",
+  "gddType": "invoke-action",
+  "gddOptions": {
+    // [mandatory] The function to invoke in the template
```

Comment:
The description of the "invoke" field should clarify if this is "the name of the function to invoke" or "an invocation of the function", i.e. - for a function called `myCustomHello` should this be `myCustomHello` or `myCustomHello("someArg0")`? I was confused by what this is.

Reply:
Good point!
I've improved the description a tad now.
The `invoke` property is supposed to contain the script being executed in the target template,  `myCustomHello("someArg0")`

Reply:
@jstarpl would you say this description is good enough?
https://github.com/SuperFlyTV/GraphicsDataDefinition/pull/11/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R386

Reply:
The way the AMCP command [CG INVOKE](https://github.com/CasparCG/help/wiki/AMCP-Protocol#cg-invoke) is documented, it does not allow to have arguments. Will that be changed?

Reply:
Yeah, the AMCP documentation definitely needs updating because AFAIK it this been supported for years. It's essentially executing any JS that it gets, not necessarily just invoking a function. You can do `CG 1-1 INVOKE 1 "var a = 1;a++;console.log(a)"` and it'll work too

Reply:
Hmmm, but what you explain is just kind of an inline function, that sits in place of the function name, like CG 1-1 INVOKE 1 "function_name". That is no contradiction to the documentation. What @nytamin and @jstarpl are talking about is having a function name and an argument, like we have for the CG UPDATE command. And that is undocumented and should, in my opinion be avoided.